Tests/InternalTests/PresenceDataTests.swift

```diff
@@ -5,15 +5,33 @@ import UIKit
 
 class PresenceDataTests: XCTestCase {
     func testSerializationPublisher() throws {
-        let data = PresenceData(type: .publisher)
-        let jsonString = try data.toJSONString()
+        var data = PresenceData(type: .publisher, rawLocations: true)
+        var jsonString = try data.toJSONString()
+        XCTAssertNotNil(jsonString)
```

Comment:
Why are you only checking if the `jsonString` is not null? I think we should be able to check if it has been parsed to a string in a correct format :thinking:

Reply:
5ac7ec651a4ee2c6d8b00e11fc4e66ed6abe5bc9